Parsers/ASimAuthentication/Parsers/vimAuthenticationSentinelOne.yaml

```diff
@@ -0,0 +1,194 @@
+Parser:
```

Comment:
same comments applicable for vim as provided in asim parser

Reply:
Done